docs/source/concept_models/vivarium_sanofi_multiple_myeloma/concept_model.rst

```diff
@@ -184,10 +184,10 @@ To answer this question, we will gather data from the 2019 Global Burden of Dise
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
   - Location: United States
-  - Cohort type: Prospective open cohort of 15 years and older
+  - Cohort type: Prospective closed cohort of individuals 15 years and older
   - Size of largest starting population: 100,000 simulants
-  - Time span: Jan 1, 2021 to Dec 31, 2025
-  - Time step: X days (e.g. 28 days) to capture time interval of clinical MM to RRMM and duration of treatment
+  - Time span: Jan 1, 2018 to Dec 31, 2025
```

Comment:
we may need to start even earlier to let the simulants "burn-in"; if we are reporting on a sim from Jan 1, 2021 to Dec 31, 2025, let's start running on Jan 1, 2011.

Reply:
Added this in. Thank you!